docs/src/pages/components/backdrop/backdrop.md

```diff
@@ -16,3 +16,12 @@ In its simplest form, the backdrop component will add a dimmed layer over your a
 ## Example
 
 {{"demo": "pages/components/backdrop/SimpleBackdrop.js"}}
+
+## Unstyled
```

Comment:
Should the header include the name of the component?

Reply:
Agree, no point in repeating the component in my opinion